api/renter.go

```diff
@@ -232,12 +239,19 @@ func (api *API) renterContractsHandler(w http.ResponseWriter, _ *http.Request, _
 	contracts := []RenterContract{}
 	for _, c := range api.renter.Contracts() {
 		contracts = append(contracts, RenterContract{
-			EndHeight:       c.EndHeight(),
-			ID:              c.ID,
-			NetAddress:      c.NetAddress,
-			LastTransaction: c.LastRevisionTxn,
-			RenterFunds:     c.RenterFunds(),
-			Size:            c.LastRevision.NewFileSize,
+			DownloadSpending: c.DownloadSpending,
+			EndHeight:        c.EndHeight(),
+			Fees:             c.TxnFee,
```

Comment:
Updated, but the new output doesn't seem to add up.

Before fix:
```
Total cost:        29.84 SC (Fees: 10.24 SC)
Funds Allocated:   19.6 SC
```

After fix:
```
Total cost:        29.84 SC (Fees: 33.84 SC)
Funds Allocated:   29.84 SC
```
(i.e. Fees > TotalCost, thus TotalCost.Sub(Fees) < 0 and the subtraction results in the smaller of the two values). 

Just as a sanity check:
Total cost is the overall cost of the File Contract including upload/download spending storage spending etc. + fees.
Remaining funds is the difference between the Transaction Inputs and Outputs allocated as of the last transaction revision. Is this data part of the Transaction or the File Contract? Or is there a different Outputs field in a File Contract?

Do you know what needs to be changed here?